addons/account/migrations/12.0.1.1/openupgrade_analysis.txt

```diff
@@ -117,178 +117,13 @@ account      / res.company              / invoice_reference_type (selection): NE
 account      / res.company              / qr_code (boolean)             : NEW 
 account      / res.company              / transfer_account_code_prefix (char): NEW 
 ---XML records in module 'account'---
-DEL account.financial.report: account.account_financial_report_assets0 (noupdate)
```

Comment:
It looks like all these removed entries should be kept. If this `DEL` entry is removed here, it means that the xml id should still be present in the module in Odoo 12 but I don't see it.

Reply:
Yeah, that' why I suspected it's not correct.